test/cli.js

```diff
@@ -2,7 +2,8 @@
 
 var child_process = require('child_process');
 
-describe('cli', function() {
+describe('cli', /* @this */ function() {
+	this.timeout(3000);
```

Comment:
These tests were being a bit flaky, sometimes being under the 2000 default and sometimes a bit over